Tests/ActionsTests/FastlaneTests.swift

```diff
@@ -6,3 +6,11 @@ class FastlaneTests: XCTestCase {
         XCTAssertTrue(true)
     }
 }
+
+#if os(Linux)
```

Comment:
So instead of adding these to each test file I actually have sourcery setup to autogenerate these extensions in the LinuxMain.swift.  You can generate the new LinuxMain.swift by running `sourcery` in the root of the repository 👍

Reply:
Ah, great idea!